src/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface ChromaticConfig {
+  chromatic: ChromaticParameters;
```

Comment:
Nesting our config options under the `chromatic` key means we can't override a subset in a test like this:
`test.use({ chromatic: { disableE2EAutoCapture: false } });`

Playwright doesn't merge option values, so that will replace the entire value of `chromatic` with a new object with only that single `disableE2EAutoCapture` key, thereby losing any other options that are set in the top-level config.

We may need to move our options out of the `chromatic` key and to the top-level.